### PR TITLE
OrbxSimulationSystems.OrbxCentral version 4.1.42

### PR DESCRIPTION
**Pull request by asmitde**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59368)

### CODE REVIEW

File: manifests/o/OrbxSimulationSystems/OrbxCentral/4.1.42.821/OrbxSimulationSystems.OrbxCentral.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: OrbxSimulationSystems.OrbxCentral
+PackageVersion: 4.1.42.821
```
Comment (Trenly) [inline manifests/o/OrbxSimulationSystems/OrbxCentral/4.1.42.821/OrbxSimulationSystems.OrbxCentral.installer.yaml:5]:
```suggestion
PackageVersion: 4.1.42
```

File: manifests/o/OrbxSimulationSystems/OrbxCentral/4.1.42.821/OrbxSimulationSystems.OrbxCentral.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: OrbxSimulationSystems.OrbxCentral
+PackageVersion: 4.1.42.821
```
Comment (Trenly) [inline manifests/o/OrbxSimulationSystems/OrbxCentral/4.1.42.821/OrbxSimulationSystems.OrbxCentral.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.1.42
```

File: manifests/o/OrbxSimulationSystems/OrbxCentral/4.1.42.821/OrbxSimulationSystems.OrbxCentral.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: OrbxSimulationSystems.OrbxCentral
+PackageVersion: 4.1.42.821
+PackageLocale: en-US
+Publisher: Orbx Simulation Systems Pty Ltd
+PackageName: Orbx Central
+License: Copyright © 2021 Orbx Simulation Systems Pty Ltd
```
Comment (Trenly) [inline manifests/o/OrbxSimulationSystems/OrbxCentral/4.1.42.821/OrbxSimulationSystems.OrbxCentral.locale.en-US.yaml:9]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

File: manifests/o/OrbxSimulationSystems/OrbxCentral/4.1.42.821/OrbxSimulationSystems.OrbxCentral.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: OrbxSimulationSystems.OrbxCentral
+PackageVersion: 4.1.42.821
```
Comment (Trenly) [inline manifests/o/OrbxSimulationSystems/OrbxCentral/4.1.42.821/OrbxSimulationSystems.OrbxCentral.yaml:5]:
```suggestion
PackageVersion: 4.1.42
```

**Review by Trenly — CHANGES_REQUESTED** [review]
The PackageVersion must match control panel; When updating this, you will need to update the name of the version folder also. You can do this in GitHub web by pressing <kbd>.</kbd>

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**